pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1458,6 +1458,12 @@ type GatewaySpec struct {
 	// +nullable
 	// +optional
 	HostNetwork *bool `json:"hostNetwork,omitempty"`
+
+	// Whether rgw dashboard is enabled for the rgw daemon. If not set, the rgw dashboard will be enabled.
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +nullable
+	// +optional
+	Dashboard *bool `json:"dashboard,omitempty"`
```

Comment:
```suggestion
	DashboardEnabled *bool `json:"dashboardEnabled,omitempty"`
```

Reply:
Rgw is an option of dashboard. 
For example, someone may choose more useful tools which similar to rgw dashboard and don't want dashboard' users modifying rgw user through dashboard.

Reply:
yes